app/js/site.js

```diff
@@ -29,12 +29,14 @@ function getRandomIntInclusive(min, max) {
 function initalizeFirebase() {
 	
   // Initialize Firebase
```

Comment:
We have opened a new google account for all of us (comp3111helloworld) so that we can access the firebase without using different google ac.